tests/tilesystem/test_tilesystem.py

```diff
@@ -1,8 +1,9 @@
 import sys
+import os
 import unittest
 from unittest import TestCase
 
-sys.path.append('../src')
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../src'))
```

Comment:
This lines up with how the other tests run, and was mildly more convenient when I was testing the change. Can be reverted if preferred.